src/relay/transforms/device_planner.cc

```diff
@@ -22,9 +22,6 @@
  * \brief Determines a unique \p VirtualDevice to hold the result of every Relay sub-expression.
  * This pass can be run multiple times, and can be run both before and after lowering.
  *
- * TODO(mbs): Rename VirtualDevice |-> VirtualDevice, and use 'virtual device' (or just 'device')
- * throughout.
- *
```

Comment:
@mbs-octoml I assume this is ok to remove.

Reply:
Oh yeah, thanks.